examples/droid/README_train.md

```diff
@@ -38,6 +38,12 @@ Run training:
 uv run --group rlds scripts/train.py pi0_fast_droid_finetune --exp-name=my_experiment --overwrite
 ```
 
+By default, training uses a simple filtering scheme that removes any frames that have little-to-no movement in the first half of the action chunk. Alternatively, you can use a custom filtering scheme by providing a json that maps from episode keys to a list of time step ranges (denoted as a tuple of start and end time step indicies) in that episode you wish to keep. The episode key is a unique ID defined as `f"{recording_folderpath}--{file_path}"`. We choose this convention because both paths are easily accessible in the DROID RLDS episodes' metadata.
```

Comment:
I think the default is actually no filtering now -- I think this is fine. I will push a follow up PR that makes your filter default. pls adjust this text tho